src/components/Nav.js

```diff
@@ -18,14 +20,54 @@ const Nav = () => {
       alignSelf="center"
       gap="medium"
     >
-      <RoutedAnchor
-        path="/"
-        icon={<GrommetIcon size="large" />}
-        label={size !== 'small' && <Text size="xlarge">grommet</Text>}
-      />
+      <Box
+        direction="column"
+        justify="between"
+        align="start"
+        alignSelf="center"
```

Comment:
Do we need the justify, align, and alignSelf props? or can we remove.

Reply:
Removed!